app/backend/wells/views.py

```diff
@@ -632,3 +634,35 @@ def lithology_geojson(request):
             get_env_variable('S3_WELL_EXPORT_BUCKET'),
             'api/v1/gis/lithology.json')
         return HttpResponseRedirect(url)
+
+
+@api_view(['GET'])
+def well_licensing(request):
+    url = 'https://test.j200.gov.bc.ca/ws/' \
+          'RestGWellsInterface/wells/{}'.format(request.GET.get('well_tag_number'))
+    headers = {
+        'content_type': 'application/json',
+        'AuthUsername': get_env_variable('E_LICENSING_AUTH_USERNAME'),
+        'AuthPass': get_env_variable('E_LICENSING_AUTH_PASSWORD')
+    }
+    response = requests.get(url, headers=headers)
+    try:
+        licence = response.json()[-1]  # Use the latest licensing value
```

Comment:
@stephenhillier I'm so tempted to update our wells licensing status here.... but that would be a crazy band-aid.... right?

Reply:
Depends on where else it's visible in the app and if there would be syncing issues. If you can eliminate any out of sync scenarios then I'd say we could do the update.